subsys/bluetooth/host/hci_core.c

```diff
@@ -3627,6 +3627,11 @@ int bt_enable(bt_ready_cb_t cb)
 	return 0;
 }
 
+bool bt_is_ready(void)
+{
+	return atomic_test_bit(bt_dev.flags, BT_DEV_READY);
```

Comment:
Is this really the right flag to check for and not BT_DEV_ENABLE? The `bt_disable()` API being added in #42409 doesn't seem to touch BT_DEV_READY at all. (ping @erwango )

Reply:
`BT_DEV_ENABLE` is set as practically the first action of `bt_enable`. The Bluetooth system is not ready to perform any actions at that point. My view is that external users want to check if they can actually use Bluetooth, not if someone has called `bt_enable`.